blockchain/blockchain.go

```diff
@@ -1434,3 +1443,33 @@ func (b *Blockchain) GetBlockByNumber(blockNumber uint64, full bool) (*types.Blo
 func (b *Blockchain) Close() error {
 	return b.db.Close()
 }
+
+// CalculateBaseFee calculates the basefee of the header.
+func (b *Blockchain) CalculateBaseFee(parent *types.Header) uint64 {
```

Comment:
There is not explanation here nor in the RFC on how is this base fee being calculated. Please provide comments here and in the RFC.

Reply:
There is an explanation under Blockchain section in RFC with the link to original source.